02_find_user_country_by_ip/src/services/detectLocation.ts

```diff
@@ -0,0 +1,24 @@
+import {ipRanges} from "../index";
+
+export const ipToNumber = (ip: string): number => {
+    const parts = ip.split('.').map(Number);
+    return parts.reduce((acc, part) => acc * 256 + part, 0);
```

Comment:
I like to avoid [magic numbers](https://eslint.org/docs/latest/rules/no-magic-numbers), leave it on your own, but it's good practice

Reply:
I support this convention, it's really useful for making the code more readable.